app/screens/DebugScreen.tsx

```diff
@@ -21,7 +21,7 @@ export const DebugScreen: FC<StackScreenProps<AppStackParamList, "Debug">> = ()
     LeftActionComponent: <BackButton />,
     title: translate("debugScreen.title"),
   })
-  const [fcmToken, setFcmToken] = React.useState<string | null>(null)
+  const [fcmToken, setFcmToken] = React.useState<string | undefined>(undefined)
```

Comment:
Why `undefined` and not `null`?

Reply:
line 56, where `fcmToken` is consumed is in a `Text` and that text accepts either a `string` or `undefined`. changing it to `null` would throw an error on that line